templates/block/block--inline-block--article-teasers-by-topic.html.twig

```diff
@@ -25,23 +25,29 @@
  * @see template_preprocess_block()
  */
 #}
+{% if is_admin %}
+<div{{ attributes }}>
+{% endif %}
 {% set classes = ['card-deck--landing', 'card-deck--landing--simple', 'card-deck--landing--x2'] %}
 <div{{ attributes.addClass(classes).setAttribute('role', 'presentation') }}>
-  {{ title_prefix }}
+  {# Need to leave the suffix in here as Layout Builder uses it for the 'configure' link #}
```

Comment:
Good comment here. Surprising LB latches onto that variable... would have been nicer of them to have a dedicated variable for that in case you want or need to move it around independently of the title.